e3sm_to_cmip/__main__.py

```diff
@@ -651,25 +651,30 @@ def _run_info_mode(self):  # noqa: C901
         messages = []
 
         # if the user just asked for the handler info
-        if self.freq == "mon" and not self.input_path and not self.tables_path:
+        # TONY SAYS: Why does this mode even exist?  And why does it depend upon "freq = mon"?
+        if self.freq == "mon" and not self.input_path and not self.tables_path: # info mode 1
```

Comment:
Carried over from original, non-refactored code (with poor documentation).

https://github.com/E3SM-Project/e3sm_to_cmip/blob/e95a74bb03bee034dd7f6fd3050f3a354944a4bf/e3sm_to_cmip/util.py#L288-L301

I'm guessing this is the most basic version of "info" mode using the default `freq=mon"` that just prints out all of the defined handlers in `e3sm_to_cmip` based on the user-selected variable list.
https://github.com/E3SM-Project/e3sm_to_cmip/blob/e95a74bb03bee034dd7f6fd3050f3a354944a4bf/e3sm_to_cmip/util.py#L159-L164